app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+###
+#  User class for authenticating via WebAuth
+###
+class User < ActiveRecord::Base
```

Comment:
There is not a defined use case that would require storage of users at this time, so let me try switching this away from being an ActiveRecord object. (And for that matter, the database in production will need to be a Symphony Oracle table for other reasons that we can discuss later.)